ir_api/scripts/acquisition.py

```diff
@@ -16,6 +16,25 @@
 LOCAL_SCRIPT_DIR = "ir_api/local_scripts"
 
 
+def _get_latest_commit_sha() -> Optional[str]:
+    """
+    Get the latest commit sha of the autoreduction-script repository
+    :return: (str) - the commit sha
+    """
+    try:
+        logger.info("Getting latest commit sha for autoreduction-script repo")
+        response = requests.get(
+            "https://api.github.com/repos/interactivereduction/autoreduction-scripts/commits/HEAD", timeout=30
+        )
+
+        return response.json()["sha"] if response.ok else None
```

Comment:
If this fails, is it possible to use the cached one?/is there a cached one?

Reply:
We don't cache a response for this. I also felt it would be more appropriate to save `None` instead of possibly the incorrect sha, in the rare case the script would be outdated in between

Reply:
The logic from my side here:

We cache the script, why do we not at the same time cache the script, and provide the sha as a cache? that way we can respond when Github is unstable, and we can only update it when we update the script? Does that not make sense?

Reply:
Changed it in a similar way to how we spoke.